dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_23__Add_skipIndividualAnalytics_column_to_program_attributes.sql

```diff
@@ -0,0 +1,8 @@
+
+-- Step 1: Add the new column with a default value of false
+ALTER TABLE program_attributes
+    ADD COLUMN IF NOT EXISTS skipanalytics boolean DEFAULT FALSE;
+
+-- Step 2: Update existing rows to ensure they all have the value false
```

Comment:
Why is this step 2 necessary?
If there was no column before, than it just do nothing.
If there was a column before and some values were `true`, why are we changing them to `false`?

Reply:
I added it as a safeguard statement, but after testing, I realized we don't actually need it.